### PR TITLE
Use latest kubevirt release

### PR DESCRIPTION
**Pull request by iranzo**

### CODE REVIEW

File: _includes/scriptlets/quickstart_minikube/04_setenv_version.sh [hunk 1]
```diff
@@ -1 +1,2 @@
-export KUBEVIRT_VERSION="v0.15.0"
+export KUBEVIRT_VERSION=$(curl -s https://api.github.com/repos/kubevirt/kubevirt/releases/latest|jq '.tag_name'|tr -d '"')
```
Comment (codificat) [inline _includes/scriptlets/quickstart_minikube/04_setenv_version.sh]: the `tr` isn't needed, just use `jq -r .tag_name`.

However, this assumes you have `jq` installed, and maybe not everyone does. In particular, maybe our lab test envs don't...

Comment (mmazur) [reply _includes/scriptlets/quickstart_minikube/04_setenv_version.sh]: > $(curl -s https://github.com/kubevirt/kubevirt/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")

Comment (mmazur) [reply _includes/scriptlets/quickstart_minikube/04_setenv_version.sh]: For reference, this is what we use: https://github.com/kubevirt/ansible-kubevirt-modules/blob/master/cluster/sync.sh#L12

**Comment by tripledes** [comment]
@iranzo @codificat there's a global variable that can be referenced {site.kubevirt_version} which is defined on _config.yml. I think it could be used for that. Downside is that we, somehow, need to update that on each release and currently there's no automation in place for that.

File: _includes/scriptlets/quickstart_minikube/04_setenv_version.sh (hunk 1 above)
Comment (tripledes) [reply _includes/scriptlets/quickstart_minikube/04_setenv_version.sh]: Python is widely installed:

```console
$ curl -s https://api.github.com/repos/kubevirt/kubevirt/releases/latest|python -c 'import sys, json; print json.load(sys.stdin)["tag_name"]'
v0.17.1
```

Tho, being picky, we're assuming it's there but we're also assuming our users are running some sort of Unix OS...so not sure where to draw the line.

**Comment by karmab** [comment]
the real issue is that latest release doesn't always match to what one would expect, only to latest published release, which could be a bug fix from an older version.

File: _includes/scriptlets/quickstart_minikube/04_setenv_version.sh (hunk 1 above)
Comment (iranzo) [reply _includes/scriptlets/quickstart_minikube/04_setenv_version.sh]: Moved to use python

**Comment by iranzo** [comment]
> @iranzo @codificat there's a global variable that can be referenced {site.kubevirt_version} which is defined on _config.yml. I think it could be used for that. Downside is that we, somehow, need to update that on each release and currently there's no automation in place for that.

With a GH token we can create a regular travis job that grabs updates and pushes as part of the build process, but that would be a different thing to handle and consider

**Comment by karmab** [comment]
let s update the site to 0.18.0 for the time beeing and treat possible automation of the version retrieval in a different PR, if needed

**Comment by iranzo** [comment]
Updated to 0.18.0

**Pull request merged**